libs/client-shared/src/lib/i18n.ts

```diff
@@ -3,6 +3,10 @@ import { Observable, of } from 'rxjs';
 
 export type Langs = 'de' | 'en' | 'fr' | 'it' | 'rm';
 
+export function isSupportedLang(lang: string): lang is Langs {
+  return ['de', 'en', 'fr', 'it', 'rm'].includes(lang);
```

Comment:
Is there a simple way of doing this without having to dublicate the array?

ChatGPT suggests this:
```

export const supportedLangs = ['de', 'en', 'fr', 'it', 'rm'] as const;

export type Langs = typeof supportedLangs[number];

export function isSupportedLang(lang: string): lang is Langs {
  return supportedLangs.includes(lang as Langs);
}

```